AlwaysOnKotlin/build.gradle

```diff
@@ -34,7 +34,7 @@ subprojects {
             targetExclude("$buildDir/**/*.kt")
             targetExclude("bin/**/*.kt")
 
-            ktlint(libs.versions.ktlint.get()).userData(["max_line_length" : "100"])
+            ktlint(libs.versions.ktlint.get())
```

Comment:
Was this the syntax to override it?

Reply:
Yep, not now results in https://github.com/android/wear-os-samples/runs/7776351662?check_suite_focus=true

> UserData should not contain '.editorconfig' properties [max_line_length]. Such properties should be passed via the 'ExperimentalParams.editorConfigOverride' field. Note that this is only required for properties that (potentially) contain a value that differs from the actual value in the '.editorconfig' file.